app/views/schedule_rules/_schedule_rule_fields.html.haml

```diff
@@ -15,15 +15,19 @@
       = f.label :end_time
       = time_select24(f, :end, hours: (0..24), minute_step: @product.reserve_interval)
 
-%h3
-  = t("views.schedule_rules.discount")
+%h3= t("views.schedule_rules.discount")
 
-= f.association :price_group_discounts do |pgd|
+%p= t("views.schedule_rules.discount_hint")
+
+= f.association :price_group_discounts, label: "hello", hint: "hi" do |pgd|
   = pgd.label pgd.object.price_group.name
-  = pgd.hint :discount_percent
-  = pgd.number_field :discount_percent, size: 4, label: false
+  = pgd.number_field :discount_percent, size: 4
   = pgd.hidden_field :price_group_id, value: pgd.object.price_group.id
   = "%"
+  - if pgd.object.price_group == PriceGroup.base 
+    %p= t("views.schedule_rules.base_discount_hint")
+  - elsif pgd.object.price_group == PriceGroup.external
+    %p= t("views.schedule_rules.external_discount_hint")
```

Comment:
Can you see if `pgd.hint :base_discount_hint` would work here?  You may need to move the copy back to `simple_form_en.yml` under hints/schedule_rule/base_discount_hint.